src/aat/resources/features/F-065/F-065.feature

```diff
@@ -68,3 +68,34 @@
       Then a negative response is received
       And the response [has an HTTP-401 Unauthorized]
       And the response has all other details as expected
+
+    @S-580
+    Scenario:Must return the case view with only the respective fields with READ access for Solicitors
```

Comment:
could you please provide space before Must

Reply:
Done.